packages/dataviews/src/field-types/index.tsx

```diff
@@ -15,8 +15,17 @@ export default function getFieldTypeDefinition( type?: FieldType ) {
 		return integer;
 	}
 
-	// If no type found, the sort function doesn't do anything.
 	return {
 		sort: () => 0,
+		isValid: ( value: any, elements?: Option[] ) => {
```

Comment:
I'm wondering about the `elements` argument here. I'm not sure it makes sense for all types and some types might have their own specific config (min, max for integer for instance...), so how can we make this more generic and that works for all types.

Reply:
Maybe we can pass the field as an argument instead.

Reply:
Yeah, agree. I made a mental note about this but forgot to implement it, this is what I had in mind https://github.com/WordPress/gutenberg/pull/64164/commits/d13edc0bd849710afc6fb4e69523e4fa581e2ee6